src/phoenix/server/api/queries.py

```diff
@@ -87,7 +88,13 @@ async def users(
             last=last,
             before=before if isinstance(before, CursorString) else None,
         )
-        stmt = select(models.User).order_by(models.User.email)
+        stmt = (
+            select(models.User)
+            .join(models.UserRole)
+            .where(models.UserRole.role != "SYSTEM")
+            .order_by(models.User.email)
+            .options(joinedload(models.User.role))
```

Comment:
```suggestion
            select(models.User)
            .where(models.UserRole.role != "SYSTEM")
            .order_by(models.User.email)
            .options(joinedload(models.User.role))
```

`joinedload` creates its own join I believe. Otherwise, we might be issuing two joins.

Reply:
I see, thanks